packages/pluggable-widgets-tools/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mendix/pluggable-widgets-tools",
-  "version": "9.20.0",
+  "version": "9.23.0",
```

Comment:
Note to self: this will become 9.24 instead.

Reply:
The version is now set at `9.24.0-beta.1` for this branch, it should/will be changed to 9.24.0 when we are close to merging/releasing.

Reply:
Done